spec/rails_helper.rb

```diff
@@ -41,11 +41,11 @@ def create_items
   end
 
   def create_user
-    let!(:user_one) { User.create(username: 'rossedfort', password: 'password', first_name: 'Ross', last_name: 'Edfort', email_address: 'rossedfort@yahoo.com', phone_number: '908-698-9024') }
+    let!(:user_one) { User.create(username: 'rossedfort', password: 'password', first_name: 'Ross', last_name: 'Edfort', email_address: 'rossedfort@yahoo.com', phone_number: '908-698-9024', role: 0) }
   end
 
   def create_admin
-    let!(:admin_one) { User.create(username: 'admin', password: 'admin', role: 1) }
+    let!(:admin_one) { User.create(username: 'admin', password: 'admin', first_name: 'tyler', last_name: 'komoroske', email_address: 'tjk@yahoo.com', phone_number: '518-339-1255', role: 1) }
```

Comment:
Line is too long. [190/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.